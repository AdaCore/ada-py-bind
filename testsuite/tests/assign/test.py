import gen

p = gen.Shape()

for x, y in zip(range(1, 20), range(1, 40, 2)):
    p.position.x = x
    p.position.y = y
    print p.position.x
    print p.position.y

