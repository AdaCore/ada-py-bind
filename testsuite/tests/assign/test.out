1
1
2
3
3
5
4
7
5
9
6
11
7
13
8
15
9
17
10
19
11
21
12
23
13
25
14
27
15
29
16
31
17
33
18
35
19
37
